test/CheckedC/dump-dataflow-facts.c

```diff
@@ -0,0 +1,256 @@
+// Tests for dumping of datafow analysis for collecting facts
+//
+// RUN: %clang_cc1 -fcheckedc-extension -fdump_extracted_comparison_facts %s 2>1 | FileCheck %s
```

Comment:
-fcheckedc-extension needs to be removed from the entire codebase as it does not do anything. Please remove it from here.

Reply:
Done.